gems/sorbet-runtime/lib/types/configuration.rb

```diff
@@ -84,7 +84,7 @@ def self.inline_type_error_handler=(value)
   end
 
   def self.inline_type_error_handler(error)
-    if @inline_type_error_handler
+    if defined?(@inline_type_error_handler) && @inline_type_error_handler
```

Comment:
instead of using `defined?` can we make sure it's always defined?

Reply:
Sure, I can do that refactor too.

Reply:
Done!